### DISCUSSION TITLE
Advanced Search filters for Task List status in Issues

### DISCUSSION THREAD

**Discussion by WardenDrew** [community/community#7475] [Projects and Issues]
Having the ability to search issues based on whether they:
- Have a task list or not
- Have a task list that is fully completed
- Have a task list that has tasks yet to be completed

Would be incredibly useful. The list of issues already conveys this information on each issue, but there is not way to filter it:
![image](https://user-images.githubusercontent.com/32055840/140798315-f4f06f49-ef05-436d-87ae-b7d58a9dbf09.png)

**Comment by nexushoratio** [comment]
I came searching through all discussions for the word *task* just to find this message.

**Comment by nexushoratio** [comment]
Partial duplicate:
- #4276
